Api/src/routes/partyRoute.js

```diff
@@ -0,0 +1,28 @@
+const express = require("express");
+const partyRouter = express.Router();
+
+partyRouter.use('/')
+  .all((req, res, next) => {
+    res.statusCode = 200;
+    res.setHeader('Content-Type', 'application/json');
+    next();
+  })
+  .get((req, res, next) => {
```

Comment:
Everything seems to be handled by `/`. This shouldn't be the case rigght? I'm sure the spec document should specify the routes?